src/styles/PostDropDown.scss

```diff
@@ -0,0 +1,11 @@
+$dropdown-divider-color:#C1C1C1;
```

Comment:
put spaces after the colon to keep the style consistent. There are few instances of that in this file. Maybe in the future we can get ESlint to cover sass files too.

Reply:
done